x-pack/plugins/apm/server/tutorial/envs/on_prem.ts

```diff
@@ -33,18 +34,10 @@ import {
 } from '../../../common/tutorial/instructions/apm_server_instructions';
 
 export function onPremInstructions({
-  errorIndices,
-  transactionIndices,
-  metricsIndices,
-  sourcemapIndices,
-  onboardingIndices,
+  apmConfig,
   isFleetPluginEnabled,
 }: {
-  errorIndices: string;
-  transactionIndices: string;
-  metricsIndices: string;
-  sourcemapIndices: string;
-  onboardingIndices: string;
+  apmConfig: APMConfig;
```

Comment:
Simplified this. Instead of passing the indicies individually I chose to pass the full config

Reply:
Would it be useful to use `Pick<APMConfig, 'apm_oss.errorIndices' | ...>` etc here? I can imagine it's going to make unit testing a little harder, e.g. you have to mock / type cast the entire `APMConfig` here. If you use `Pick`, you can still pass in the entire APMConfig object, but you can't use other properties (or at least TypeScript will complain about it), but in tests, you won't have to supply other (required properties).

Reply:
Sure, I can do that